README.md

```diff
@@ -192,33 +183,23 @@ When instantiating a Distortable Image, pass in a `fullResolutionSrc` option set
 
 ```JS
 img = L.distortableImageOverlay('example.png', {
-  corners: [
-    L.latLng(51.52,-0.14),
-    L.latLng(51.52,-0.10),
-    L.latLng(51.50,-0.14),
-    L.latLng(51.50,-0.10)
-  ],
   fullResolutionSrc: 'large.jpg'
 }).addTo(map);
-
-L.DomEvent.on(img._image, 'load', img.editing.enable, img.editing);
 ```
 
 ### Suppress Toolbar
 
 `suppressToolbar` (*optional*, default: false, value: *boolean*)
 
-To initialize an image without its toolbar, pass it `suppressToolbar: true`. 
+To initialize an image without its toolbar, pass it `suppressToolbar: true`.
 
 Typically, editing actions are triggered through our toolbar interface or our predefined keybindings. If disabling the toolbar, the developer will need to implement their own toolbar UI or just use the keybindings. (WIP API for doing this)
 
 This option will override other options related to the toolbar, such as [`selected: true`](#Selected)
 
-## Setup - Multiple Image Interface
-
-1.  From the root directory, run `npm install` or `sudo npm install`
+## Multiple Image Interface
 
-2.  Open `examples/select.html` in a browser (todo -- add gh pages demo)
+(todo -- add gh pages demo)
```

Comment:
Oh, does select.html not work for this anymore?

Reply:
oh haha it does that was silly of me I'll remove the todo